calico-cloud/get-started/connect/operator-checklist.mdx

```diff
@@ -404,7 +404,7 @@ kubectl get tigerastatus
 | 2   | calico                        | TRUE      | FALSE       | FALSE    | 11m   |
 | 3   | cloud-core                    | TRUE      | FALSE       | FALSE    | 11m   |
 | 4   | compliance                    | TRUE      | FALSE       | FALSE    | 9m39s |
-| 5   | intrusion-detection            | TRUE      | FALSE       | FALSE    | 9m49s |
+| 5   | intrusion-detection           | TRUE      | FALSE       | FALSE    | 9m49s |
```

Comment:
here's an example of a whitespace correction -- note that I'm well aware that the whitespace isn't relevant, but overall this repository seems to take great care to properly align content (there were a very small handful of cases I ran into from when I initially started that weren't in sync, but with the couple of commits between when I started and when I synced, there were suddenly quite a few).